packages/component/src/Styles/StyleSet/SuggestedAction.js

```diff
@@ -18,8 +18,16 @@ export default function createSuggestedActionStyle({
   suggestedActionDisabledBorderWidth,
   suggestedActionDisabledTextColor,
   suggestedActionHeight,
+  suggestedActionsStackedLayoutButtonTextWrap,
+  suggestedActionsStackedLayoutButtonMaxHeight,
   subtle
 }) {
+  const patchedMinHeight =
+    suggestedActionsStackedLayoutButtonTextWrap && suggestedActionHeight < suggestedActionsStackedLayoutButtonMaxHeight
```

Comment:
could you use standard if..else to make it more readable ?

Reply:
erm, @compulim tends to go for ternary. @compulim thoughts?

Reply:
Actually nevermind; it's now a simple ternary.

Reply:
Is this one replaceable somehow by `Math.min()`?

I am also confused why `suggestedActionsStackedLayoutButtonTextWrap` is only needed for `suggestedActionHeight < suggestedActionsStackedLayoutButtonMaxHeight`, but not the vice versa?